tests/componenttests/client/tests/base/MediaKeysTestMethods.cpp

```diff
@@ -60,14 +60,23 @@ const std::string kInvalidKeySystem{"notExpected"};
 const std::vector<std::string> kKeySystems(firebolt::rialto::server::kSupportedKeySystems.begin(),
                                            firebolt::rialto::server::kSupportedKeySystems.end());
 const std::string kVersion{"123"};
+constexpr int32_t kMediaSourceTypeUnknown{0};
```

Comment:
Should use enums for  kMediaSourceTypeUnknown, kMediaSourceTypeAudio & kMediaSourceTypeVideo (probably dont need to use variables for them, just use the enums)